docs/code-examples/point2d_random.py

```diff
@@ -0,0 +1,17 @@
+"""Log some random points with color and radii."""
+from __future__ import annotations
```

Comment:
```suggestion
```
These only add noise and no value for these examples. Definitely need to go. None of these code examples even contain type annotations

Reply:
This allows to use modern type hints all the way back to 3.8. PyUpgrade (via `just py-format`) will ensure that *if* that thing is there, which is why now ruff enforces it.

Reply:
I agree that it's annoying here, though.

Reply:
`# isort: skip_file` should disable the rule for the example but we don't what that from showing either. Any means of dropping top-level comments when building the page?

Reply:
@jleibs I figured it out. Add the following in `rerun_py/pyproject.toml`, adjusting the path to where those example files are:

```toml
[tool.ruff.per-file-ignores]
"path/to/doc/examples/*" = ["I002"]
```

Reply:
Nice! thanks.